db/dialect_clickhouse.go

```diff
@@ -2,15 +2,76 @@ package db
 
 import (
 	"context"
+	"database/sql"
 	"fmt"
+	"sort"
 	"strings"
 
 	"github.com/streamingfast/cli"
 	sink "github.com/streamingfast/substreams-sink"
+	"go.uber.org/zap"
 )
 
 type clickhouseDialect struct{}
 
+func (d clickhouseDialect) Flush(tx *sql.Tx, ctx context.Context, l *Loader, outputModuleHash string, cursor *sink.Cursor) (int, error) {
```

Comment:
I would like that comment from https://github.com/streamingfast/substreams-sink-postgres/pull/35#issue-1843970287 be inserted here to explain the different batching.